composer/algorithms/stochastic_depth/stochastic_depth.py

```diff
@@ -189,17 +199,19 @@ def __init__(self,
 
     @property
     def find_unused_parameters(self) -> bool:
-        """DDP parameter to notify that parameters may not have gradients if it is dropped during the forward pass."""
+        """DDP parameter to notify that parameters may not 
+        have gradients if it is dropped during the forward pass.
+        """
 
         return (self.stochastic_method == "block")
 
     def match(self, event: Event, state: State) -> bool:
         """Run on :attr:`~composer.core.event.Event.INIT`, as well as
-    :attr:`~composer.core.event.Event.BATCH_START` if ``drop_warmup > 0``.
+        :attr:`~composer.core.event.Event.BATCH_START` if ``drop_warmup > 0``.
 
         Args:
-            event (Event): The current event.
-            state (State): The current state.
+            event (:class:`Event`): The current event.
```

Comment:
Type links not rendering. Also other algorithms don't seem to have docstrings for `match`, `apply`, and `find_unused_parameters`, so please confirm the convention with @ravi-mosaicml or @hanlint

Reply:
The original type links already render (https://docs.mosaicml.com/en/latest/api_reference/composer.algorithms.stochastic_depth.stochastic_depth.html#composer.algorithms.stochastic_depth.stochastic_depth.StochasticDepth.apply) so not changed are neccessary here.

Reply:
It should not require the `:class:` notation here. I wonder what sphinx version are you on locally? Could be helpful to start with a clean virtual environment and re-install all dependencies; would that fix it?

Reply:
@ravi-mosaicml Yes, this is what the later PRs (e.g. #1073 ) were created to address. In that PR, I reverted all of these unneeded `:class:` annotations.

Reply:
Sounds good. Going to merge this one in first, then do #1073.